modules/radsBidAdapter.js

```diff
@@ -0,0 +1,155 @@
+import * as utils from '../src/utils';
+import {config} from '../src/config';
+import {registerBidder} from '../src/adapters/bidderFactory';
+import { BANNER, VIDEO } from '../src/mediaTypes';
+
+const BIDDER_CODE = 'rads';
+const ENDPOINT_URL = 'https://rads.recognified.net/md.request.php';
+const ENDPOINT_URL_DEV = 'https://dcradn1.online-solution.biz/md.request.php';
+const DEFAULT_VAST_FORMAT = 'vast2';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['rads'],
+  supportedMediaTypes: [BANNER, VIDEO],
+  isBidRequestValid: function(bid) {
+    return !!(bid.params.placement);
+  },
+  buildRequests: function(validBidRequests, bidderRequest) {
+    return validBidRequests.map(bidRequest => {
+      const params = bidRequest.params;
+      const videoData = utils.deepAccess(bidRequest, 'mediaTypes.video') || {};
+      const sizes = utils.parseSizesInput(videoData.playerSize || bidRequest.sizes)[0];
+      const width = sizes.split('x')[0];
+      const height = sizes.split('x')[1];
+      const placementId = params.placement;
+
+      const rnd = Math.floor(Math.random() * 99999999999);
+      const referrer = encodeURIComponent(bidderRequest.refererInfo.referer);
+      const bidId = bidRequest.bidId;
+      const isDev = params.devMode || false;
+
+      let endpoint = ENDPOINT_URL;
+      if (isDev) {
```

Comment:
A ternary operator can be used here instead of assignment followed by an `if` statement